### ISSUE TITLE
`.file` creates links to files that don't exist

### ISSUE THREAD

**Issue by hadley** [r-lib/cli#587]
e.g. `cli::cli_inform("{.file doesntexist}")`

**Comment by gaborcsardi** [comment]
Yeah, that's intentional. Eg with a link to .Rprofile you can create the file if it does not exist. Idk what rstudio does in this case, though.

**Comment by hadley** [comment]
It errors when you click on it 😬

**Comment by hadley** [comment]
But even if it did do something, we need some way in cli to label relative paths that we don't expect to be clickable. It would also be nice to have a different path than what's shown (so you could provide an absolute path but show a relative path), but even then it's useful to have something that doesn't attempt to add the links.

**Comment by gaborcsardi** [comment]
Yeah, I left these for "hyperlinks, part two". There are a lot of possibilities, so won't be able to have a good DSL, but we'll have explicit function calls to cover all of them.
